docs/start/getting-started/fragments/flutter/integrate.md

```diff
@@ -115,3 +115,27 @@ void _recordEvent() async {
 ```
 
 At this point you are almost ready to run your app.  In the next section, we will use Amplify CLI to configure your backend AWS resources.
+
+## Using Flutter Hot Reload and Hot Restart
+
+Flutter offers [hot reload and hot restart](https://flutter.dev/docs/development/tools/hot-reload) functionality in order to aid the development process.
+
+A key difference between hot reload and restart is that hot restart destroys the state in dart, while hot reload does not.  Amplify-flutter should automatically re-configure your application and re-wire its plugins upon a hot restart, so there is no additional effort needed on your part.
+
+Depending on how and when you are calling `Amplify.configure`, you may wish to check whether or not Amplify has already been configured because it should not be configured more than once.  
+
+To do this, you can use Amplify-flutter's `isConfigured` getter:
+
+```dart
+  void _setConfigurationStatus_() {
+    setState(() {
+      configurationStatus = Amplify.isConfigured;
+    });
+  }
+```
```

Comment:
Why make customers maintain another flag when they can just use `Amplify.isConfigured`

Reply:
@Amplifiyer Should I just remove the code block entirely?

Reply:
yeah if we have `Amplify.isConfigured` code samples elsewhere, we can just remove it, though while reviewing @Jordan-Nelson's PR I found that we haven't updated or changed from `AmplifyAlreadyConfiguredException` to `Amplify.isConfigured`

Reply:
FWIW, a state variable is probably still useful for controlling UI but maybe we don't have to make assumptions for how a customer wants to handle that.